site/content/tutorial/05-events/03-event-modifiers/text.md

```diff
@@ -25,5 +25,6 @@ The full list of modifiers:
 * `capture` — fires the handler during the *capture* phase instead of the *bubbling* phase ([MDN docs](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events#Event_bubbling_and_capture))
 * `once` — remove the handler after the first time it runs
 * `self` — only trigger handler if event.target is the element itself
+* `trusted` - only trigger handler if event.trusted is true. Event trusted is only true, if the event is triggered by a user action.
```

Comment:
assuming this is a valid modifier, it looks like it should also be added to https://svelte.dev/docs#Element_directives ?

Reply:
resolved in https://github.com/sveltejs/svelte/pull/6566/commits/c0226184b31b4a8e9268eb7a154564aa008f6ac6

Reply:
Released in 3.39.0